### PR TITLE
[Runtime Security]Add PCCE O'Neal content

### PR DESCRIPTION
**Pull request by rcaballeromx**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=pcce-oneal-content

### CODE REVIEW

**Review by manukumar6 — APPROVED** [review]

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-00.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
+:toc: macro
+== 31.00 Release Notes
```
Comment (divyabhushan) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-00.adoc:2]: == 32.00 Release Notes

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-00.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+|December 3, 2023
+
+|Type
+|Major release
+
+|SHA-256
+|39d1f808550f7491dcc63e392b5eb55e8f9068494b4f49c07c71da17433f7396
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/31/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtimes, tools, and orchestrators.
```
Comment (divyabhushan) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-00.adoc:24]: Version number update to 32 in the link

File: docs/en/compute-edition/32/rn/release-information/known-issues-32.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
+== 31.xx Known Issues
```
Comment (divyabhushan) [inline docs/en/compute-edition/32/rn/release-information/known-issues-32.adoc:1]: @rcaballeromx did you want to update the version number to O'Neal here as well!

Comment (rcaballeromx) [reply docs/en/compute-edition/32/rn/release-information/known-issues-32.adoc:1]: Thank you Divya but I created PR #268 to address the release note updates.

**Pull request merged**